src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -381,20 +381,13 @@ def get_events_from_nodestore(
 def _make_seer_call(
     create_grouping_records_request: CreateGroupingRecordsRequest, project_id: int
 ) -> BulkCreateGroupingRecordsResponse | None:
-    try:
-        seer_response = _retry_operation(
-            post_bulk_grouping_records,
-            create_grouping_records_request,
-            retries=3,
-            delay=2,
-            exceptions=Exception,
-        )
-    except Exception as e:
```

Comment:
removing this, as `post_bulk_grouping_records` function already handles exceptions and returns the failure reason back